src/main/scala/com/fulcrumgenomics/util/Metric.scala

```diff
@@ -176,6 +178,22 @@ trait Metric extends Product {
   /** Get the values of the arguments in the order they were defined. */
   def values: Seq[String] = productIterator.map(formatValues).toSeq
 
+  /** Gets the value of the field by name. */
+  def apply(name: String): Option[String] = {
+    this.names.indexOf(name) match {
+      case -1   => None
+      case idx  => Some(this.values(idx))
+    }
+  }
+
+  /** Gets the value for the ith field. */
+  def apply(i: Int): Option[String] = {
```

Comment:
Could this be private please?  I think it's fine to do internally, but I don't think it should be part of the public API as the _only_ safe and reliable way to get the index is via scanning the names, and this might encourage the idea that the indices are durable and safe to use.